modules/extension/xsd/xsd-gml3/src/test/java/org/geotools/gml3/bindings/MultiPolygonTypeBindingTest.java

```diff
@@ -69,4 +71,25 @@ public void testEncode() throws Exception {
         assertEquals("geometry.1", getID(children.item(0)));
         assertEquals("geometry.2", getID(children.item(1)));
     }
+
+    @Test
+    public void testEncodePolygon() throws Exception {
+        Geometry geometry = GML3MockData.polygon();
+        geometry.setUserData(CRS.decode("EPSG:4326"));
+        GML3EncodingUtils.setID(geometry, "geometry");
+        Document dom = encode(geometry, GML.MultiPolygon);
+        print(dom);
```

Comment:
print should probably be commented
```suggestion
        // print(dom);
```